Lecture-14/src/main/java/factory/ApacheHttpResponseFactory.java

```diff
@@ -0,0 +1,61 @@
+package factory;
+
+import constant.HttpMethodType;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+import static util.PrintHelper.printInfoAboutIoSyntaxException;
+import static util.PrintHelper.printInfoAboutUriSyntaxException;
+
+public class ApacheHttpResponseFactory {
+
+    private static final String TOKEN = "9831d26fead1dcb4cd9419a9107a84555742f0a3";
+    private static final CloseableHttpClient httpClient = HttpClients.custom().build();
+
+    private ApacheHttpResponseFactory() {
+    }
+
+    public static CloseableHttpResponse generateRequest(String url, HttpMethodType methodType) {
+        switch (methodType) {
+            case GET: {
```

Comment:
`{` braces are not needed in `case` statements

Reply:
done